ui/components/multichain/asset-picker-amount/max-clear-button/index.scss

```diff
@@ -0,0 +1,4 @@
+.max-clear-button {
+  float: right;
```

Comment:
Floats were a godsend back in the day but they are pretty outdated. If you need to send something to the far right I would suggest giving the parent a `display="Display.Flex` and the child a `marginLeft="auto"` That should send the max clear button to the far right